lg_earth/src/lg_earth/kmlsync.py

```diff
@@ -330,10 +330,24 @@ class KmlQueryHandler(tornado.web.RequestHandler):
     def initialize(self):
         self.playtour = self.application.playtour
         self.playtour_service = self.application.playtour_service
+        self.planet = self.application.planet
+        self.planet_service = self.application.planet_service
+        self.get_planet = self.application.get_planet
+
+    def _wait_for_planet(self, planet):
```

Comment:
there's absolutely no guarantee that when `_wait_for_planet` succesfully finishes then the planet is really changed. We're observing kmlsync http server that's polled by GE from time to time.